qiskit/transpiler/target.py

```diff
@@ -287,6 +290,7 @@ def __init__(
                 matches the qubit number the properties are defined for. If some
                 qubits don't have properties available you can set that entry to
                 ``None``
+            meas_map(list, dict): List of sets of qubits that must be measured together.
```

Comment:
The two representations of `meas_map` are accepted as input, but only one is documented here.

Reply:
So far, most part of the `meas_map` type is list and this is converted into Dict in inner codes. However, I think that 'meas_map' accepts Dict, e.g. macros.measure, in some cases because Dict of `meas_map` is easy to understand intuitively rather than List if there exist hardware constraints.

I treated the `meas_map` type as Dict and implemented macros.measure in backendV2. (It seems that format_measmap was implemented because it is easier to use for Dict than for List.)
So, if the input is a List, I would like to use format_measmap to convert the List to a Dict, and accept both List and Dict as input.